### DISCUSSION TITLE
Blinds does not give silver S/SS

### DISCUSSION THREAD

**Discussion by molneya** [ppy/osu#28188] [Ideas]
Silver S/SS is given when a mod that obscures gameplay is active such as Hidden, Flashlight or FadeIn (https://osu.ppy.sh/wiki/en/Gameplay/Grade). Blinds should probably also fall into this category, and perhaps some of the other new lazer mods too.

Example score: https://osu.ppy.sh/scores/2838536729 (even if its hard to see, trust me, its a normal gold S)

**Comment by bdach** [comment]
I guess this makes sense, it's just I don't know where the line is drawn. Is something like No Scope visibility reduction too? What else?

**Reply by molneya** [reply]
the way i see it, there are 3 categories of obscuring gameplay:

- mods that definitely remove or obscure gameplay elements (blinds, bubbles, no scope, traceable)
- mods that remove gameplay elements but less drastically than hidden (grow, deflate)
- mods that don't remove elements but obscure gameplay in a different way (barrel roll, transform, wiggle, freeze frame)

the first category should definitely all be given silver grades. i separated the second category from the first since i can see arguments for why something less obscuring than hidden should not be given silver grades. the third category has no clear analogue to anything that currently gives silver grades so this could be up for debate. for all other mods, i dont see why they should be given silver grades at all (unless i missed something).

**Reply by emi-x** [reply]
I would not put No Scope in the same category as silver-S mods

It can definitely be harder for people who play mouse or 'regular' tablets, but for people who play on touch devices or display tablets, the No Scope mod does not change visibility in a way that i think it deserves any extra flair. 

I main No Scope now since I play on display tablet, and I would be adamantly opposed to anything 'extra' being given for that mod since it would be unfair to others if I could get something without any more work simply because my playstyle makes the mod ultimately a nonissue

**Reply by Tyfaspo** [reply]
Friend who's been playing with traceable skin in stable for a long time is saying the same thing, his one of the main complaints was traceable not giving silver S despite being visual obscuring and having the same score and pp multiplier as HD, all hands for making blinds and traceable getting the same treatment as hidden of flashlight mods.

**Reply by Tenexxt** [reply]
> I would not put No Scope in the same category as silver-S mods
> 
> It can definitely be harder for people who play mouse or 'regular' tablets, but for people who play on touch devices or display tablets, the No Scope mod does not change visibility in a way that i think it deserves any extra flair.
> 
> I main No Scope now since I play on display tablet, and I would be adamantly opposed to anything 'extra' being given for that mod since it would be unfair to others if I could get something without any more work simply because my playstyle makes the mod ultimately a nonissue

For osu! (android ver. atleast), on standard osu, you are given automatically a 'touch device' mod, so making it work how incompatible mods work, exept of making the 100% exeptional (like, how there CAN'T be a flashlight + blinds), make it that the 'touch device' device mod and 'no scope' work the same, BUT they can be put together exept you get a unranked score.
That would be a hard fix, a easy way to fix this would be just deleting 'no scope' from mods menu, so you cant pick it, but not compleatly deleting it from the code (replays would break asf)
